tests/test_estimate_emotions.py

```diff
@@ -46,10 +46,12 @@ def test_estimate_emotions_as_dict(self):
         """
         Test emotion estimator 'asDict' method
         """
-        faceDetection = self.detector.detectOne(EMOTION_IMAGES["fear"])
-        warp = self.warper.warp(faceDetection)
-        emotionDict = self.emotionEstimator.estimate(warp.warpedImage).asDict()
-        self.validate_emotion_dict(emotionDict)
+        for emotion in ALL_EMOTIONS:
+            with self.subTest(image_emotion=emotion):
```

Comment:
For what? It is  test of a method asDict only/

Reply:
removed